management/azure_mgmt_resources/spec/resource_groups_spec.rb

```diff
@@ -126,36 +126,11 @@
     end
   end
 
-  it 'should list resources of resource group' do
```

Comment:
did the operation list_by_resource_group move from resource_groups to resources in the spec?
It'd be good to note these type of changes down in the change log/release notes.